### ISSUE TITLE
Packet loss with last stable ntopng community 3.6.181004

### ISSUE THREAD

**Issue by malvivent7** [ntop/ntopng#2083]
Hi, i have installed ex-novo ntopng stable on centos 7 using instructions on ntopng site. The server machine is an intel s2600jf 24xeon cores with 128GB of ram with an i350 dualnetwork card My startup ntopng command parameters is:
ntopng --max-num-hosts "600000" --http-port "3000" --https-port "0" --dns-mode "3" --local-networks "xxx.xxx.xxx.xxx/24 ecc." --traffic-filtering "httpbl:ljfiunmrxmbf" --community 
Network-flow is captured on any interface but eno2 is the interface that receive tapflow from my core hp-switch (1Gbit network port) I see that eno2 has lot of packet drops often up to 49%.
What could i investigate to low this issue? I think that my system could achieve and sustain more traffic

**Comment by simonemainardi** [comment]
Every interface is handled by a single-thread in ntopng. So it doesn't matter if you've a xeon with 24 cores.

Without PF_RING Zero Copy, I wouldn't expect ntopng to be able to handle a lot more than 1Mpps per thread. However, you can try and run pfcount on interface eno2 to see if you experience drops also with that.

There are several ways to increase the capture speed, though. They include PF_RING Zero Copy and Receive Side Scaling (RSS). This post explains what you can do to tune the installation: https://www.ntop.org/ntopng/best-practices-for-running-ntopng/

So basically I would try, in order:
- PF_RING (if you are not yet using it)
- PF_RING Zero Copy
- RSS

Let us know.

**Comment by malvivent7** [comment]
Thanks Simone; as you pointed me on the right direction now i have 4 queues with no packetloss with zc... so far so good!! I have a question it's possible to switch to italian language and not english on the community version of ntopng?

**Comment by simonemainardi** [comment]
Awesome, thanks for reporting!

As per the question on the language, note that multi-lang support is a pro-only feature.
